src/bindings/python/src/openvino/runtime/ie_api.py

```diff
@@ -2,7 +2,8 @@
 # Copyright (C) 2018-2023 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from typing import Any, Iterable, Union, Optional
+from __future__ import annotations
```

Comment:
Why are future features needed, what is the reason? Isn't `"CompiledModel"` good enough?

Reply:
I just like it to be in one style :)
Otherwise, returning type for `clone` and `deepcopy` also should be take into quotes.

Reply:
removed __future__